Sources/Components/MotorSidebar/Models/MotorSidebarView+ViewModel.swift

```diff
@@ -0,0 +1,137 @@
+import UIKit
+import FinniversKit
+
+extension MotorSidebarView {
+    public class ViewModel {
+        public let mainSections: [Section]
+        public let secondary: Section?
+
+        public init(mainSections: [Section], secondary: Section? = nil) {
+            self.mainSections = mainSections
+            self.secondary = secondary
+        }
+    }
+}
+
+extension MotorSidebarView.ViewModel {
+    public class Section {
+        public var isExpanded: Bool
+        public let isExpandable: Bool
+        public let ribbon: Ribbon?
+        public let price: Price?
+        public let header: Header?
+        public let content: [Body]
+        public let bulletPoints: [String]
+        public let buttons: [Button]
+
+        public init(
+            isExpandable: Bool,
+            isExpanded: Bool = true,
+            ribbon: Ribbon? = nil,
+            price: Price? = nil,
+            header: Header? = nil,
+            content: [Body],
+            bulletPoints: [String] = [],
+            buttons: [Button] = []
+        ) {
+            self.isExpandable = isExpandable
+            self.isExpanded = isExpanded
+            self.ribbon = ribbon
+            self.price = price
+            self.header = header
+            self.content = content
+            self.bulletPoints = bulletPoints
+            self.buttons = buttons
+        }
+    }
+
+    public struct Ribbon {
```

Comment:
Just to check if you saw it, we already have `RibbonView` in FinniverseKit and it has `Style` enum for all possible styles equal colors. So, at least, you can reuse it )